pkg/apis/batch/v1alpha1/job.go

```diff
@@ -81,8 +78,11 @@ type VolumeSpec struct {
 	// not contain ':'.
 	MountPath string `json:"mountPath" protobuf:"bytes,1,opt,name=mountPath"`
 
+	// defined the PVC name
+	VolumeClaimName string `json:"volumeClaimName,omitempty" protobuf:"bytes,2,opt,name=volumeClaimName"`
```

Comment:
Why would u add this? Would prefer auto gen it.

Reply:
Yeah, there would be a few codes which can be optimized, would like to do it when the initial sync patch get merged.